src/helpers/SCSBApiHelper.js

```diff
@@ -267,8 +267,8 @@ const SCSBApiHelper = module.exports = {
     }
 
     // Extract Patron Barcode
-    if (object.patronInfo && object.patronInfo.barCode) {
-      scsbModel.patronBarcode = object.patronInfo.barCode;
+    if (object.patronInfo && Array.isArray(object.patronInfo.barCodes) && object.patronInfo.barCodes.length) {
+      scsbModel.patronBarcode = object.patronInfo.barCodes[0]; // obtain first barcode item in array
```

Comment:
Ugh, the current logic in the BarcodeService—and I'm not saying this is the best logic—is to use the last barcode. I'm pretty sure initially used the first barcode and then we got some complaints. FWIW, we haven't gotten any complaints since switching to the _last_ barcode.

Reply:
Hi,

Updated logic to use last barcode.

Reply:
PLEASE APPROVE IF IT MEETS STANDARDS! 👍